tree/dataframe/src/RLoopManager.cxx

```diff
@@ -203,6 +205,25 @@ static void ThrowIfNSlotsChanged(unsigned int nSlots)
    }
 }
 
+/**
+\struct MaxTreeSizeRAII
+\brief Scope-bound change of `TTree::fgMaxTreeSize`.
+
+This RAII object stores the current value result of `TTree::GetMaxTreeSize`,
+changes it to maximum at construction time and restores it back at destruction
+time. Needed for issue #6523 and should be reverted when #6640 will be solved.
+*/
+struct MaxTreeSizeRAII {
+   Long64_t fOldMaxTreeSize;
+
+   explicit MaxTreeSizeRAII() : fOldMaxTreeSize(TTree::GetMaxTreeSize())
```

Comment:
afaik `explicit` does not do anything here

```suggestion
   MaxTreeSizeRAII() : fOldMaxTreeSize(TTree::GetMaxTreeSize())
```

Reply:
I thought it made the code cleaner, but yes probably there won't be any actual case where this does something.